### ISSUE TITLE
Set custom date format to any Clock skin.

### ISSUE THREAD

**Issue by milosic5** [HanSolo/Medusa#180]
Is there any method to set custom date format of any Clock skin?

**Comment by HanSolo** [comment]
do you need a specific format or dependent on the locale? Setting the locale should have an effect as far as I remember...will check at the weekend

**Comment by milosic5** [comment]
Yes, I need specific custom date format which is not dependent on locale. For example instead of dots to put any sign or to remove seconds or hours or mins and for (YYYY-DD-MM) to be custom visible etc.

**Comment by HanSolo** [comment]
Ok so could add a custom date formatstring property where you can pass the formatString e.g. „dd.MM.YYYY“ and another property to switch the use of this customDateFormatString on/off. 
In a first step you then could create your own skin where you make use of the customDateFormatDateString. Your custom skin can be a fork from an existing skin with additional functionality. If you tell me the skin where you need the behavior I could also add it to this skin so that you don‘t need to fork an existing skin.
